src/components/filtermenu/filtermenu.js

```diff
@@ -51,12 +65,12 @@ define(['require', 'dom', 'focusManager', 'dialogHelper', 'loading', 'apphost',
         });
 
         apiClient.getFilters(filterMenuOptions).then(function (result) {
```

Comment:
You can avoid the `const self = this;` by using an arrow function, here.
```suggestion
        apiClient.getFilters(filterMenuOptions).then((result) => {
```